src/Microsoft.Identity.Web/Resource/MicrosoftIdentityIssuerValidatorFactory.cs

```diff
@@ -15,9 +17,20 @@ namespace Microsoft.Identity.Web.Resource
     /// </summary>
     internal class MicrosoftIdentityIssuerValidatorFactory
     {
+        public MicrosoftIdentityIssuerValidatorFactory(
+            IOptions<AadIssuerValidatorOptions> aadIssuerValidatorOptions,
+            IHttpClientFactory httpClientFactory)
```

Comment:
Did you need to add a package dependency for this? You at least need to register it in DI.

Reply:
no need to add a package dependency. IHttpClientFactory is already used by Microsoft.Identity.Web

Reply:
right, what @jmprieur said.